### PR TITLE
fix(tokens): bundle pkg

### PR DESCRIPTION
**Pull request by kyranjamie**
Already published, not sure on status of auto-publish?

This PR builds the token package, needed for use outside monorepo. [Adds some configs having read this guide.](https://adamcoster.com/blog/pnpm-config)

### CODE REVIEW

**Comment by pete-watters** [comment]
I'm not sure what's happening with the CI here. 

Auto publish is working but on `release`. So once this is merged if you manually create a release it will then release all the updated packages

**Comment by pete-watters** [comment]
> I see. Any reason not to release from main like we do on extension?

We do release from `main`. Once this gets merged to `dev` we then create a release of the newest stuff on `dev` and that gets merged to `main` and auto-releases the packages, creating release notes etc. 

We can add improvements if we want to automate it better. Packages should probably auto release once merged to `dev`

File: packages/bitcoin/package.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
   "version": "0.0.1",
   "type": "module",
   "author": "Leather.io contact@leather.io",
-  "type": "module",
```
[the remark this answers is not in the document]
Comment (kyranjamie) [reply packages/bitcoin/package.json:7]: it was duplicated, see L5

File: packages/tokens/package.json [hunk 2]
```diff
@@ -1,10 +1,12 @@
 {
   "name": "@leather-wallet/tokens",
   "description": "Shared design tokens",
-  "version": "0.0.10",
+  "version": "0.0.13",
   "author": "Leather.io contact@leather.io",
   "scripts": {
-    "typecheck": "tsc --noEmit -p ./tsconfig.json"
+    "build": "tsc",
```
[the remark this answers is not in the document]
Comment (kyranjamie) [reply packages/tokens/package.json]: done

**Review by fbwoolf — APPROVED** [review]

**Review by alter-eggo — APPROVED** [review]

File: packages/ui/package.json [hunk 3]
```diff
@@ -6,13 +6,13 @@
     "build-storybook:web": "storybook build -c ./src/.storybook-web",
     "lint": "eslint .",
     "lint:fix": "pnpm lint --fix",
-    "prepare": "panda codegen -c ./panda.config.ts",
+    "panda": "panda codegen -c ./panda.config.ts",
```
Comment (pete-watters) [inline packages/ui/package.json:9]: Good idea renaming this 👍

File: packages/ui/src/theme-web/typography.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
-import { getExtensionTextVariants } from '@leather-wallet/tokens';
+import { getWebTextVariants } from '@leather-wallet/tokens';
```
Comment (pete-watters) [inline packages/ui/src/theme-web/typography.ts:1]: This is going to be a breaking change in the extension so we should release and update it soon

File: packages/ui/package.json (hunk 3 above)
Comment (kyranjamie) [reply packages/ui/package.json:9]: Not just rename, `prepare` default runs on `pnpm i` which creates a problem on install as it requires there to be a build first

Comment (pete-watters) [reply packages/ui/package.json:9]: WOW, OK, good catch then!

**Pull request merged**